docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,377 @@
+   -  LS-SR71
+   -  Nanostation XM
+   -  Nanostation Loco XM
+   -  Picostation
+
+-  Unknown
+
+   -  A5-V11
+
+-  VoCore
+
+   -  VoCore (8M, 16M)
+
+Atheros target migration
+------------------------
+
+All Atheros MIPS devices built with the ``ar71xx-generic``,
+``ar71xx-nand`` as well as ``ar71xx-tiny`` were deprecated upstream and
+are therefore not available with Gluon anymore.
+
+Many devices previously built with ``ar71xx-generic`` and
+``ar71xx-nand`` are now available with the ``ath79-generic`` as well as
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+-  8Devices
+
+   -  Carambola 2
+
+-  Aerohive
+
+   -  HiveAP 121
+
+-  Allnet
+
+   -  ALL0315
+
+-  Buffalo
+
+   -  WZR-HP-G300NH2
+   -  WZR-HP-G450H
+
+-  GL.iNet
+
+   -  6408A v1
+
+-  NETGEAR
+
+   -  WNDR3800
+   -  WNDR4300
+   -  WNDRMAC
+   -  WNDRMAC v2
+
+-  OCEDO
+
+   -  KOALA
+
+-  TP-Link
+
+   -  WR2543
+
+- Ubiquiti
+
+   -  Rocket
+
+-  WD
+
+   -  MyNet N600
+   -  MyNet N750
+
+-  ZyXEL
+
+   -  NB6616
+   -  NB6716
+
+Features
+--------
+
+Wireguard
+~~~~~~~~~
+
+Gluon got WireGuard support. This allows offloading **encrypted**
+connections into kernel space, increasing performance by forwarding
+packets without the need for context switches between user and kernel
+space.
+
+In order to reuse existing (already verified) fastd-keypairs for
+WireGuard, a key derivation procedure is `currently being
+developed <https://github.com/freifunk-gluon/gluon/pull/2601>`__. This
+should ease migration from fastd to WireGuard in case whitelisting VPN
+keys is desired.
+
+fastd L2TP
+~~~~~~~~~~
+
+fastd can now act as a connection broker for unencrypted L2TP-based
+tunneling within Gluons mesh-vpn framework.
+
+In addition to a sufficiently configured fastd-based VPN server, this
+requires further modifications to a site. :ref:`VPN fastd methods`
+
+Major changes
+-------------
+
+OpenWrt
+~~~~~~~
+
+This release is based on the newest OpenWrt 22.03 release branch.
+It ships with Linux kernel 5.10 as well as wireless-backports 5.15.
+
+
+Network changes (DSA / Upgrade-Behavior)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ramips-mt7621 and lantiq-xrx200 targets now use the upstream DSA
+subsystem instead of OpenWrt swconfig for managing ethernet switches.
+
+Gluon detects the existing user-intent and automatically applies it over
+to DSA syntax. See the section about network reconfiguration for more
+details.
+
+System reconfiguration
+~~~~~~~~~~~~~~~~~~~~~~
+
+The network and system-LED configurations are now re-generated after
+each update / invocation of ``gluon-reconfigure``.
+
+The user-intent is preserved within Gluon’s implemented functionality
+(Wired-Mesh / Client access / WAN).
+
+As an additional feature, Gluon now supports assigning roles to
+interfaces. This behavior is explained here –>
+https://github.com/freifunk-gluon/gluon/pull/2584/files
```

Comment:
I tried to propose a link but without rebasing this PR it just doesn't make sense. But here should be the corresponding docs section which is now merged.

Reply:
now you can

Reply:
proposing changes to other files is to my knowledge not possible so: https://github.com/blocktrron/gluon/pull/9